### DISCUSSION TITLE
NEW FEATURE REQUEST

### DISCUSSION THREAD

**Discussion by GrassIsNotBlue** [RedShyGuy/Vapecord-ACNL-Plugin#116] [Ideas]
1: if I close acnl and play again next day, I need to select and enable all my cheats again. What if you could make a backup/restore for the favourite cheats list or even the whole plugin menu.
2: backups for placed items per room. It would be awesome to be able to save/restore all furniture placements in my room/whole house, or the tree/bush/rock/flower..-positions of my "hacker garden" on the island.
3: paths should be placed on bridges
4: real-time PWP placement with the PWP-list you see when at the desk in the town hall
5: paths should be placed in front of the front doors of houses without getting removed the next startup, this also with trees/bushes right next to houses and trees against the fence by the train track.
If any of these features could be possible, then thank you so much!

**Comment by GrassIsNotBlue** [comment]
* never mind nr 3, it already works, there was just something wrong last time I checked

**Reply by SirEnder125** [reply]
Number 1 already exists as part of CTRPF itself.
